src/LinearMap.jl

```diff
@@ -333,7 +334,7 @@ A list of vertices.
 We assume that the underlying set `X` is polyhedral.
 Then the result is just the linear map applied to the vertices of `X`.
 """
-function vertices_list(lm::LinearMap{N})::Vector{Vector{N}} where {N<:Real}
+function vertices_list(lm::LinearMap{N}; remove_redundant::Bool=true)::Vector{Vector{N}} where {N<:Real}
```

Comment:
Can we rename `remove_redundant` to `prune`? This is shorter and used for the H-polyhedron already. I just checked and we do not have `remove_redundant` elsewhere.